src/Scrollable/Scrollable.scss

```diff
@@ -0,0 +1,71 @@
+@import "variables";
+
+.pgn__scrollable-body {
+    overflow-x: hidden;
```

Comment:
Is this needed?

Reply:
the overflow-x : hidden? I thought it was what fixed that horizontal scroolbar but I can double check it still needed

Reply:
The horizontal scrollbar should be fixed in a different way. It's possible some combo of width setting + padding makes the child element wider than the parent.

Reply:
Ah I see, so yeah given more horiz spacing perhaps I can avoid the scrollbar that way, will try

Reply:
Interesting, I now removed all margins for the shadow, and now I don't se the horiz scrollbar either, even withou overflow-x:hidden so I think we are good?